tests/unit/test_cube.py

```diff
@@ -470,18 +471,183 @@ def and_it_accepts_a_JSON_format_cube_response(self, cube_response, expected_val
+                                            {"name": "B", "alias": "B"},
+                                        ]
+                                    },
+                                }
+                            }
+                        },
+                    }
+                },
+            ),
+        ),
+    )
+    def it_knows_its_cube_dict_composition(
+        self, _cube_response_prop_, cube_response, expected_value
+    ):
+        _cube_response_prop_.return_value = cube_response
+        cube = Cube(None)
+
+        cube_dict = cube._cube_dict
+
+        assert cube_dict == expected_value
+
+    @pytest.mark.parametrize(
+        "cube_response, expected_value",
+        (
+            ({}, []),
+            ({"foo": "bar"}, []),
+            (
+                {
+                    "result": {
+                        "measures": {
+                            "mean": {"metadata": {"type": {"subvariables": ["A", "B"]}}}
+                        }
+                    }
+                },
+                ["A", "B"],
+            ),
+        ),
+    )
+    def it_knows_its_mean_subvariables(
+        self, _cube_response_prop_, cube_response, expected_value
+    ):
+        _cube_response_prop_.return_value = cube_response
+        cube = Cube(None)
+
+        mean_subvariables = cube._mean_subvariables
+
+        assert mean_subvariables == expected_value
+
+    @pytest.mark.parametrize(
+        "cube_response, expected_value",
+        (
+            ({}, []),
+            ({"foo": "bar"}, []),
+            (
+                {
+                    "result": {
+                        "measures": {
+                            "mean": {
+                                "metadata": {
+                                    "references": {
+                                        "subreferences": [
+                                            {"name": "A", "alias": "A"},
+                                            {"name": "B", "alias": "B"},
+                                        ]
+                                    }
+                                }
+                            }
+                        }
+                    }
+                },
+                [{"name": "A", "alias": "A"}, {"name": "B", "alias": "B"}],
+            ),
+        ),
+    )
+    def it_knows_its_mean_subreferences(
+        self, _cube_response_prop_, cube_response, expected_value
+    ):
+        _cube_response_prop_.return_value = cube_response
+        cube = Cube(None)
+
+        mean_subreferences = cube._mean_subreferences
+
+        assert mean_subreferences == expected_value
+
+    @pytest.mark.parametrize(
+        "fixture, expectation",
+        (
+            (CR.MEANS_CAT_X_CAT_HS, None),
+            (CR.MR_X_CAT_HS, None),
+            (NA.NUM_ARR_MEANS_NO_DIMS, ["0001", "0002"]),
+        ),
+    )
+    def it_knows_if_it_has_means_subvariales(self, fixture, expectation):
```

Comment:
This doesn't seem like a unit test. It uses fixtures and the mechanism to load them from the file.